LECT_2/Lect2_homework/job1/helpers.py

```diff
@@ -0,0 +1,42 @@
+import os
+import shutil
+import json
+import requests
+
+AUTH_TOKEN = os.environ.get("API_AUTH_TOKEN")       # Secret token from the environment
+URI = 'https://fake-api-vycpfa6oca-uc.a.run.app/'   # Path to API
+
+
+def get_sales(date, raw_dir=''):
+    page = 1
+    actual_raw_dir = os.getcwd().replace('\\job1', '') + '\\file_storage' + raw_dir
+
+    if os.path.exists(actual_raw_dir):
+        shutil.rmtree(actual_raw_dir)
+
+    os.makedirs(actual_raw_dir)
+
+    while True:
+        response = requests.get(
```

Comment:
Please, take a look how to pass GET parameters to `requests.get`. You should not build the full URL manually, just pass them in `params` variable.